packages/components/src/VisuallyHidden/VisuallyHidden.tsx

```diff
@@ -0,0 +1,25 @@
+import { ReactNode, createElement, HTMLAttributes } from "react"
+import { OverrideClassName } from "~types/OverrideClassName"
+import styles from "./VisuallyHidden.module.scss"
+
+export type AllowedTags = "div" | "span"
+
+export type VisuallyHiddenProps = {
+  children: ReactNode
+  /**
+   * The HTML tag rendered by this component
+   */
+  tag?: AllowedTags
```

Comment:
Would be good to have a short description in the docs describing what this does.